packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -76,7 +74,7 @@ def config_all_loggers():
 
 def set_logging_handler(
     logger: logging.Logger,
-    formatter_base: Optional[Logging] = None,
+    formatter_base=None,
```

Comment:
why is this annotation gone?

Reply:
Aha! There is a bug in pylint and it seems I simply remove it instead of ignoring that ... will add  it back again